coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/parsetree/ParseTreeBuilder.java

```diff
@@ -876,6 +890,113 @@ protected SqlNode visitCTE(ASTNode node, ParseContext ctx) {
     return result;
   }
 
+  @Override
+  protected SqlNode visitWindowSpec(ASTNode node, ParseContext ctx) {
+    // See Apache Hive source code: https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/parse/CalcitePlanner.java#L4339
+    // "private RelNode genSelectForWindowing(QB qb, RelNode srcRel, HashSet<ColumnInfo> newColumns)"
+
+    // Hive Antlr ASTNode Tree:
```

Comment:
Can we write a representative/comprehensive enough SQL and its corresponding full dump?

Reply:
Added the SQL example